### ISSUE TITLE
Test Open Library's Color Contrast

### ISSUE THREAD

**Issue by bpmcneilly** [internetarchive/openlibrary#4522]
For this task, we want you to take a look through all of the Open Library content & make sure the text has sufficient color contrast.

### Steps

1. Go through each of the pages under the 'Pages to Test' heading
    1. If you can't take on every page, no worries! Just add a comment here & let us know what pages you're reviewing ☺️
2. Using a color contrast tool of your choosing (see the "Resources" heading below) measure the contrast of any new color combination
    1.  Generally, the color contrast ratio of text should be 4.5:1 or higher
        1.  If you select two colors in your checker, a ratio should be displayed - choosing white (`#ffffff`) and black (`#000000`), for instance, will create a contrast ratio of 21.0:1
        2. Some "Large Text" (that's at least 14 Point & Bolt or 18 Point) can be a lower ratio of 3.0:1
3. If any color contrast ratios are too low, please file a bug (or better yet a PR!) related to this image 
    1. General bug / PR formatting is available on the main Accessibility Issue: #4202 
    2. For contrast issues / PRs, make sure you include what the current contrast ratio is & what you think alternative colors could be (and what those contrast ratios would be)

### Resources
* Color contrast tools are available are available in a variety of formats:
    * Firefox includes a color contrast tool from its [developer tools](https://developer.mozilla.org/en-US/docs/Tools/Accessibility_inspector#Color_contrast)
    * Web based contrast tools, such as [WebAIM's Color Contrast Checker](https://webaim.org/resources/contrastchecker/) exist
    * Some desktop apps, such as the [Colour Contrast Analyser](https://github.com/ThePacielloGroup/CCAe) also exist
* If you have any questions on other color contrast tools, how to interpret results or anything else feel free to the accessibility contact! 
    * Contact info is available on the main Accessibility Issue: #4202

### Pages to test

- [x] [Home](https://openlibrary.org/)
- [x] [Subjects](https://openlibrary.org/subjects/)
- [x] [Individual Subject](https://openlibrary.org/subjects/architecture)
- [x] [Lists](https://openlibrary.org/lists)
- [x] [Individual List](https://openlibrary.org/people/mekBot/lists/OL103955L/Childhood_Favorites)
- [x] [Advanced Search](https://openlibrary.org/advancedsearch)
- [x] [Search Results](https://openlibrary.org/search?q=The+Crucible&mode=ebooks&has_fulltext=true)
- [x] [Search Inside Results](https://openlibrary.org/search/inside?q=The+Crucible&mode=ebooks&has_fulltext=true)
- [x] [Book Page](https://openlibrary.org/works/OL66347W/The_Crucible)
- [x] [Help Page](https://openlibrary.org/help/faq/about)
- [x] [Library Explorer](https://openlibrary.org/explore)
- [x] [Log In](https://openlibrary.org/account/login)
- [x] [Loans](https://openlibrary.org/account/loans)
- [x] [Want To Read](https://openlibrary.org/people/bpmcneilly/books/want-to-read)
- [x] [Want To Read 'Stats'](https://openlibrary.org/people/bpmcneilly/books/want-to-read/stats)
- [x] [Privacy Settings](https://openlibrary.org/account/privacy)
- [x] [Edit Profile](https://openlibrary.org/people/bpmcneilly?m=edit)
- [x] [Report a Problem](https://openlibrary.org/contact?path=/people/bpmcneilly)
- [x] [Recent Changes](https://openlibrary.org/recentchanges)
- [x] [Add a Book](https://openlibrary.org/books/add)

**Comment by Yashs911** [comment]
0 Contrast Errors on [Subjects](https://openlibrary.org/subjects) Page. https://wave.webaim.org/report#/https://openlibrary.org/subjects

**Comment by Yashs911** [comment]
4 Contrast Errors on Individual Subject Page.

Contrast ratio due to `#738C35` is **3.79:1**
Changing it to `#687E30` will make the ratio **4.54:1**

![image](https://user-images.githubusercontent.com/64412143/107255237-195e1b80-6a5e-11eb-989f-60124b61f4ca.png)

**Comment by Yashs911** [comment]
4 Contrast Errors on Individual List Page.

Contrast ratio due to `#738C35` is 3.79:1
Changing it to `#647c36` will make the ratio 4.68:1

![image](https://user-images.githubusercontent.com/64412143/107260407-1534fc80-6a64-11eb-9ba9-2ba47c593f54.png)

**Comment by Sabreen-Parveen** [comment]
@bpmcneilly and @Yashs911 I have checked `login` and `Help` page, there are no color contrasts in them.
https://wave.webaim.org/report#/https://openlibrary.org/help/faq/about
https://wave.webaim.org/report#/https://openlibrary.org/account/login

**Comment by Yashs911** [comment]
> ### 0 Contrast Errors
> * **Subjects Page**. https://wave.webaim.org/report#/https://openlibrary.org/subjects
> * **Lists Page**. https://wave.webaim.org/report#/https://openlibrary.org/lists
> * **Library Explorer** Page https://wave.webaim.org/report#/https://openlibrary.org/explore
> * **Login Page** [#4522 (comment)](https://github.com/internetarchive/openlibrary/issues/4522#issuecomment-775866796)
> * **Help Page** [#4522 (comment)](https://github.com/internetarchive/openlibrary/issues/4522#issuecomment-775866796)
> * **Want To Read** https://wave.webaim.org/report#/https://openlibrary.org/people/yashs911
> * **Want To Read 'Stats'**
> * **Recent Changes** https://wave.webaim.org/report#/https://openlibrary.org/recentchanges
> 
> ### Fix/Fixed Contrast Issues
> * **Home Page** #4538
> * **Individual Subject Page** #4558
> * **Individual List Page** #4558
> * **Advanced Search** #4571 (Red Search)
> * **Search Results** #4580
> * **Search Inside Results** #4580
> * **Loans** #4558
> * **Privacy Settings** #4571 (Red Cancel)
> * **Edit Profile** #4571 (Red Cancel)
> * **Report a Problem** #4571 (Red Cancel)
> * **Add a Book** #4571 (Red Cancel)
> * **Book Page** #4601

@bpmcneilly I think we have checked all the mentioned pages 🎉

**Comment by bpmcneilly** [comment]
@Yashs911 - Thanks to you and @Sabreen-Parveen for all your work on this! I'm reviewing now. I'm going to add a few new accessibility issues soon & may also format things differently to make tracking a little easier. Be on the lookout 😄

**Comment by lokesh** [comment]
All work scoped here is complete.
